Spanner/tests/System/BackupTest.php

```diff
@@ -48,9 +51,9 @@ class BackupTest extends SpannerTestCase
 
     public static function setUpBeforeClass()
     {
-        if ((bool) getenv("SPANNER_EMULATOR_HOST")) {
-            return;
-        }
+        $self = new static;
+        $self->checkAndSkipEmulatorTests();
```

Comment:
Make `checkAndSkipEmulatorTests()` static (`public static function checkAndSkipEmulatorTests()`) and call it here using `self::checkAndSkipEmulatorTests()`. No need to instantiate anything. :)